api/data/NMBS/connections.php

```diff
@@ -406,43 +405,51 @@ public static function parseHafasXml($serverData, $lang, $fast, $request, $showA
     private static function addOccupancy($connections, $date)
     {
         $occupancyConnections = $connections;
+        $mongodbExists = true;
```

Comment:
Document what you're doing here in the code: say that you're going to try to connect to mongodb, but if it doesn't work, you'll just show the normal data for developers that want to add something to the iRail API without touching occupancy scores.